server/tests/models/record.js

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable max-len */
+
+import { Record } from '../../src/models/record.js';
+import { expect } from 'chai';
+import { describe, it } from 'mocha';
+
+describe('Testing Record model', function() {
+  it('1. Creating new record', function(done) {
+    const r = new Record({id: 'r1', latitude: -87.46, longitude: 180, timestamp: new Date('2022-02-22T10:08:00.000-05:00'), driver_id: 'd1', slip_score: 50.88});
+    r.validate().then(() => {
+      expect(r.id).to.equal('r1');
+      expect(r.latitude).to.equal(-87.46);
+      expect(r.longitude).to.equal(180);
+      expect(r.timestamp.valueOf()).to.equal(new Date(1645542480000).valueOf());
+      expect(r.driver_id).to.equal('d1');
```

Comment:
Can we add a test for checking if  driver_id fields are missing or empty. The driver ids are required for analytics.